V2/tests/api/azure/project_azure_endpoints_test.py

```diff
@@ -0,0 +1,254 @@
+import json
+from http import HTTPStatus
+
+import pytest
+from faker import Faker
+import azure.functions as func
+
+from time_tracker.projects._application import _projects as azure_projects
+from time_tracker.projects import _domain as domain
+from time_tracker.projects import _infrastructure as infrastructure
+
+PROJECT_URL = '/api/projects/'
+
+
+@pytest.fixture(name='insert_project')
+def insert_project(test_db, insert_customer, project_factory, customer_factory) -> domain.Project:
+    inserted_customer = insert_customer(customer_factory(), test_db)
+
+    def _new_project():
+        project_to_insert = project_factory(customer_id=inserted_customer.id)
+        dao = infrastructure.ProjectsSQLDao(test_db)
+        inserted_project = dao.create(project_to_insert)
+        return inserted_project
+    return _new_project
+
+
+def test__project_azure_endpoint__returns_all_projects(
+    insert_project
+):
+    inserted_projects = [
+        insert_project().__dict__,
+        insert_project().__dict__
+    ]
+
+    req = func.HttpRequest(method='GET', body=None, url=PROJECT_URL)
+    response = azure_projects._get_projects.get_projects(req)
+    projects_json_data = response.get_body().decode("utf-8")
+
+    assert response.status_code == HTTPStatus.OK
+    assert projects_json_data == json.dumps(inserted_projects)
+
+
+def test__project_azure_endpoint__returns_an_project__when_project_matches_its_id(
```

Comment:
the name could be test__project_azure_endpoint__returns_a_project__when_project_matches_its_id with a inside an